examples/buckgenproj_test.sh

```diff
@@ -26,11 +26,10 @@ pipenv install
 
 pipenv run $GIT_ROOT/genproj.py \
            --output_directory "$PROJECT_OUT" \
-           --buck_module_path "/mockapp" \
+           --blaze_module_path "/mockapp" \
            --gen_type flat \
            --lines_of_code 150000
 
-MOCK_WORKSPACE_PATH="$PROJECT_OUT/App/MockApp.xcworkspace"
-cd "$GENPROJ_ROOT"
-buck project "//mockapp/App:MockApp" -d
-time xcodebuild build -scheme MockApp -sdk iphonesimulator -workspace "$MOCK_WORKSPACE_PATH" > "$BUILD_LOG_PATH"
```

Comment:
I am not quite sure how this was working before. This used buck to generate an xcodeproj and then invoking an `xcodebuild` command, however, the project generated by Buck does not seem to delegate to `buck` executable itself so this was actually doing a normal `xcodebuild`. I am not sure if this was intended but I changed it here.

Reply:
Yeah that was the intention, it was to benchmark xcodebuilds, with projects generated from buck, not buck builds.   But you still can do it either way, by generating an xcodeproject and building with xcode or just building with buck directly.

Reply:
oh thanks for clarifying!